bazel/foreign_cc/llvm.patch

```diff
@@ -0,0 +1,25 @@
+# Workaround for Envoy's CMAKE_BUILD_TYPE=Bazel.
```

Comment:
We don't upstream LLVM/WAVM support, so we don't need this patch.

Reply:
Done.

Reply:
Push update to GitHub? None of the "Done" things were updated.